product_sale_price_security/views/product_views.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='group_standard_price']" position="attributes">
+                <attribute
+                    name="groups"
+                >product_sale_price_security.group_product_sale_price</attribute>
+            </xpath>
+        </field>
+    </record>
+    <record id="product_template_tree_view" model="ir.ui.view">
+        <field name="name">product.template.tree</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_tree_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='list_price']" position="attributes">
+                <attribute name="invisible">1</attribute>
```

Comment:
```suggestion
                <attribute name="groups">product_sale_price_security.group_standard_price</attribute>
```
Can you check if we can simply do this. If it's sufficient, we should update the RPL module as well.